modules/@apostrophecms/image/ui/apos/components/AposMediaManagerToolbar.vue

```diff
@@ -78,46 +97,50 @@ export default {
         status: {},
         value: { data: '' }
       },
-      filterFields: {
-        published: {
-          field: {
-            name: 'published',
-            type: 'radio',
-            label: 'Published State',
-            choices: [
-              {
-                label: 'Published',
-                value: 'published'
-              },
-              {
-                label: 'Unpublished',
-                value: 'unpublished'
-              }
-            ]
-          },
-          value: { data: 'published' },
-          status: {}
+      filterFields: [
+        {
+          name: 'published',
+          label: 'Published',
+          inputType: 'radio',
+          choices: [
+            {
+              value: true,
+              label: 'Published'
+            },
+            {
+              value: false,
+              label: 'Draft'
+            },
+            {
+              value: 'any',
+              label: 'Both'
+            }
+          ],
+          allowedInChooser: false,
```

Comment:
?

Reply:
This is copied from the backend output (`window.apos.modules['@apostrophecms/image']`). I'm not sure what that's about.